src/Apps/Artwork/Components/ImageCarousel.tsx

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components"
 
 import Icon from "Components/Icon"
 import { Arrow } from "Styleguide/Elements/Arrow"
-import { Responsive } from "Utils/Responsive"
+import { Media } from "Utils/Responsive"
```

Comment:
@alloy - would you mind reviewing this file's refactor (ImageCarousel) and seeing if this is the right approach? There are conditional elements throughout and I was able to (somewhat) cleanly refactor them into the new approach but wondering if there might be a better way or if there's something we could do at the lib level in order to avoid having to execute conditional functions within each breakpoint and pass strings. It pretty much seems ok, but we're having to hoist elements out of the main `render` in order to keep things reasonable which would be nice not to have to do. I could be missing something!

Reply:
I'm going to copy this file's changes into my current branch